lib/simp/relchecks.rb

```diff
@@ -162,7 +161,7 @@ def self.extract_version_changelog(changelog_entries, version,
   def self.load_and_validate_changelog(component_dir, verbose)
     # only get valid changelog entries for the latest version
     # (up to the first malformed entry)
-    info = Simp::ComponentInfo.new(component_dir, true)
+    info = Simp::ComponentInfo.new(component_dir, true, verbose)
```

Comment:
The reason this was intentionally not passed down is because it spews too much information.

Reply:
Users won't see verbose output unless they request it, so it's reasonable for them to expect a lot of information when they do.  In my case, it spewed too _little_ information while I was trying to trace the task's behavior.  That ended up costing more time and effort than if the variable had been passed.  

(At the moment, it's also a useless parameter―it isn't used elsewhere in the code or the tests.)